=== tests/cases/compiler/noImplicitReturnsWithProtectedBlocks3.ts ===
declare function log(s: string): void;
>log : (s: 文字) => 无值
>s : 文字
>类型别名 :=> 文字

declare function get(): number;
>get : () => 数字

function main1() : number {
>main1 : () => 数字

    try {
        return get();
>get() : 数字
>类型别名 :=> 数字
>get : () => 数字
    }
    catch(e) {
>e : 任意
>类型别名 :=> 任意

        log("in catch");
>log("in catch") : 无值
>类型别名 :=> 无值
>log : (s: 文字) => 无值
>"in catch" : "in catch"
    }
}
